terraform/aws/components/base/variables.tf

```diff
@@ -88,7 +88,7 @@ variable "docdb_master_password" {
 
 variable "docdb_tls" {
   type = string
-  default = "disabled"
+  default = "enabled"
```

Comment:
I guess better option would be to set the default to disabled (like it already is) and enable it via variable set in etc